library/src/main/java/com/airbnb/android/airmapview/MapboxWebViewMapFragment.java

```diff
@@ -22,4 +22,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     return view;
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
@felipecsl I found a [better example](https://www.mapbox.com/mapbox.js/example/v1.0.0/toggle-baselayers/) of how to do this.  It actually involves swapping the `tileLayer` instead of the `id` field.